packages/react/src/experimental/SelectPanel2/SelectPanel.tsx

```diff
@@ -309,8 +312,13 @@ const Panel: React.FC<SelectPanelProps> = ({
                 '--max-height': maxHeightValue,
                 '--position-top': `${position?.top ?? 0}px`,
                 '--position-left': `${position?.left ?? 0}px`,
+                visibility: internalOpen ? 'visible' : 'hidden',
+                display: 'flex',
```

Comment:
if we could find a way to put this in the SelectPanel.module.css file I think that'd be preferred. Could default to 'visible' and add a [data-visibility-hidden] that hides it

Reply:
I tried that, but the specificity wasn't high enough to override the default browser stylesheet, which sets `display` to `none`. We basically have no choice but to set this in the `style` attribute 😬